project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,301 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+"""
+Tests for enforcing stage‐change and creation restrictions on project tasks.
```

Ruff (0.11.9)

Docstring contains ambiguous `‐` (HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)